src/OpenTelemetry.Extensions.Hosting/OpenTelemetry.Extensions.Hosting.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <Description>OpenTelemetry hosting integration adds startup extensions to register OpenTelemetry into .NET Core applications.</Description>
+    <Description>Startup extensions to register OpenTelemetry into .NET Core applications</Description>
```

Comment:
This is not tied exclusively to .NET core applications.. Any application using the Msft.Extensions.DependencyInjection and Ms.Extensions.Hosting can leverage this..

Reply:
Do you have a suggested wording?

Reply:
Startup extensions to register OpenTelemetry into .NET applications using Microsoft.Extensions.DependencyInjection and Microsoft.Extensions.Hosting - ?